eth/backend.go

```diff
@@ -440,10 +440,6 @@ func (s *Ethereum) StartMining() error {
 			}
 			cli.Authorize(eb, wallet.SignData)
 		}
-		// If mining is started, we can disable the transaction rejection mechanism
-		// introduced to speed sync times.
-		s.handler.enableSyncedFeatures()
```

Comment:
There is no reason to enable "synced features" especially in the context of Proof-of-Stake, which only generates payload once the chain is fully synced.

Even in the legacy world, the transactions should still be rejected/mining should still be aborted before the node is synced.

I think it should be safe to remove it.